jekyll/_cci2/workflows.adoc

```diff
@@ -233,6 +236,31 @@ curl --request POST \
 ----
 --
 
+=== Cancel a job
+
+To Cancel a job follow these steps:
+
+[.tab.cancel.CircleCI_web_app]
+--
+. Select the `hold` job in the *Workflows* page of the CircleCI web app.
+. Select btn:[Cancel].
+--
+
+[.tab.cancel.API]
+--
+. Set up your API authentication. Steps are available in the xref:api-developers-guide#add-an-api-token[API developers guide].
+. You need your job ID (which is the job ID for the job requiring approval) to make the API POST request. Depending on your use case, you might be configuring an approval job in a pipeline, or using an external service to approve a job. You can get the required parameters using xref:variables#[pipeline values or environment variables], or by xref:webhooks#[receiving webhooks] or polling the API. To cancel a job, use the link:https://circleci.com/docs/api/v2/index.html#operation/cancelJobByJobID[cancel a job] endpoint, for example:
++
+[,shell]
+----
+curl --request POST \
+  --url https://circleci.com/api/v2/jobs/<job-ID>/cancel \
+  --header 'authorization: Basic REPLACE_BASIC_AUTH' \
```

Comment:
```suggestion
```
Redundant header, "Circle-Token:" is good enough

Reply:
Thank you, will remove!